### DISCUSSION TITLE
Welcome to pdfmake-wrapper Discussions!

### DISCUSSION THREAD

**Discussion by Lugriz** [Lugriz/pdfmake-wrapper#28] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  **NOTE**: Select the category according to your discussion (ideas, general, Q&A, etc...).

  To get started, comment below with an introduction of yourself and tell us about what you build using this library.

**Comment by thebirdman7** [comment]
PDFMake is great! I would love it if PDFMake would support loading SVGs from a file. We have some data sources that we load dynamically that contain images, It is a bit onerous to have to look through those dynamic sources for SVGs and then load them dynamically and adjust image tags so they are either 'svg' or 'image'. Then we also have to adjust the image dictionary to make sure we are not adding SVGs there or it will cause an error. It would be so much easier if we could just add them to the image dictionary. If you could knock this one out, it would save me some time. Thanks so much!

Another thing I would like to mention. If an image is not found, it causes an error and the entire report process is halted. I think it would be a nice idea if you had a scalable error image that showed up instead sort of like browsers do, instead of bringing the whole process to a halt.

**Reply by Lugriz** [reply]
Hi @thebirdman7!

Thank you for using pdfmake-wrapper. Those are good ideas, they will be considered in next versions. we are also open to contributions.
